internal/report/report.go

```diff
@@ -1225,13 +1230,21 @@ func computeTotal(prof *profile.Profile, value, meanDiv func(v []int64) int64) i
 		if v < 0 {
 			v = -v
 		}
-		ret += v
+		total += v
+		if sample.HasTag("pprof::base", "true") {
+			diffTotal += v
+			diffDiv += d
+		}
 		div += d
```

Comment:
I think I had this comment but it was not answered before: Assuming that it's intentional that accumulating the div is not under the tag condition, can you add a comment on why so?

Reply:
Fixed.